connection.js

```diff
@@ -454,16 +479,19 @@ class Connection {
                  }
                 // now just before TimecontrolString is being tested, we again make sure it has the latest value
                 universalMaintimeTimecontrolString = String(t.time_control);/*
-                "fischer", "byoyomi", "canadian" */
-
-                // sanity check : if not fischer, not byoyomi, not canadian
-                if ((universalMaintimeTimecontrolString !== "fischer") && (universalMaintimeTimecontrolString !== "byoyomi") && (universalMaintimeTimecontrolString !== "canadian")) {
-                    conn_log ("error, could not find time control in " + t.time_control);
-                    return { reject : true, msg: "error, could not find time control in " + t.timecontrol};
+                "fischer", "byoyomi", "canadian", "simple" */
+
+                // sanity check : if not fischer, not byoyomi, not canadian, not simple
+                // time control peculiarities : 
+                // - fischer : doesnt have a minperiods or maxperiods
+                // - simple : doesnt have a maintime, only periodtime, so we let it slide for main time reject
+                if ((universalMaintimeTimecontrolString !== "fischer") && (universalMaintimeTimecontrolString !== "byoyomi") && (universalMaintimeTimecontrolString !== "canadian") && (universalMaintimeTimecontrolString !== "simple")) {
```

Comment:
need to do that on top of #129 , after UHMAEAT 2.2

Reply:
Sure, do this later.

Reply:
would be UHMAEAT 2.3, but more importantly a function on this would be great help

remember that UHMAEAT 1.0 is same as 2.2 on this issue, so i'd just merge and look later :+1: